modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -119,7 +132,7 @@ var backendServiceInvoker = function () {
         header.setValue(constants.APPLICATION_JSON);
         httpMethodObject.addRequestHeader(header);
```

Comment:
This is the content type header for the http request.there is a design process going to enable the content type and customer header to the invoker. That might change the source but the purpose of this is to indicates the media type of the entity-body sent to the recipient